dsc/docs-conceptual/dsc-2.0/concepts/class-based-resources.md

```diff
@@ -680,22 +749,75 @@ an invalid value and how the value was invalid.
 > the rest of the DSC Resource's logic to work. If the DSC Resource creates a file at that location,
 > don't validate its existence on the property declaration.
 
-If you're using any [complex properties][19], be sure to apply validation attributes to those
+If you're using any [complex properties][22], be sure to apply validation attributes to those
 subproperties as well. Those subproperties are validated at the same time as their parent property.
 
 ### Use enums instead of ValidateSet
 
 If a property of the class-based DSC Resource has a list of valid values it accepts, define it as an
-[enum property][20] instead of using the **ValidateSet** attribute.
+[enum property][24] instead of using the **ValidateSet** attribute.
 
 This gives users a better error message and makes maintenance easier if you use those values
 anywhere else in your module. Instead of having to update every cmdlet or check for that property,
 you can update the enum definition.
 
+Remember that enums are value type properties. They default to `0` when an instance of the
+class-based DSC Resource is created. If you set an explicit default value for an enum property, that
+enum can't be unmanaged. If a user doesn't specify the property, the DSC Resource will behave the
+same as if the user had explicitly specified the default value.
+
+To support unmanaged enum properties, make sure not to define a label for the `0` value. By default,
+if you don't specify the integer value for any labels when using the `enum` statement, the first
+declared label has a value of `0`. Instead, define the first label with a value of `1` (or any
+higher value).
+
+For example:
+
+```powershell
+enum OptionalSetting {
+    FirstOption = 1
+    SecondOption
+    ThirdOption
+}
+```
+
+Then, in your **Test** and **Set** methods, you can ignore the enum property if the value is `0`.
+In **Test**, if the enum property is `0`, ignore the current state of the DSC Resource on the
+system. Don't report the DSC Resource as out of desired state if the current state is a valid value.
+In **Set**, make sure your logic for modifying system state ignores the enum property if it's `0`.
```

Comment:
Let change method names to use code markup throughout. This is one example. I didn't look for others.

```suggestion
Then, in your `Test()` and `Set()` methods, you can ignore the enum property if the value is `0`.
In `Test()`, if the enum property is `0`, ignore the current state of the DSC Resource on the
system. Don't report the DSC Resource as out of desired state if the current state is a valid value.
In `Set()`, make sure your logic for modifying system state ignores the enum property if it's `0`.
```